### DISCUSSION TITLE
Issue when Configuring superset (config.py) with LDAP

### DISCUSSION THREAD

**Discussion by aakanshajecrc** [apache/superset#18940] [Q&A / Help]
I am trying to do authentication:LDAP in superset using the guide at: 
https://flask-appbuilder.readthedocs.io/en/latest/security.html

Please note that I am attempting it on Ubuntu 20.04 VMs behind a proxy. And I am running superset on http://localhost:9000/(in dev mode)

After following all the steps successfully when I run it on 9000 port
I got this error: Invalid login. Please try again

#### How to reproduce the bug

1 Clone Superset repository
2 cd superset
3 docker-compose up
4 in another terminal(cd superset/superset-frontend)
5 npm run dev-server

### Expected results

Login Successfully.

### Actual results

Invalid login. Please try again

modify config.py
from flask_appbuilder.security.manager import AUTH_LDAP

AUTH_ROLE_ADMIN = 'Admin'
AUTH_USER_REGISTRATION = True
AUTH_USER_REGISTRATION_ROLE = "Public"
AUTH_TYPE = 2
AUTH_LDAP_SERVER = "ldap://ldapserver.new"
AUTH_LDAP_USE_TLS = False
AUTH_LDAP_SEARCH = "cn=rrscw,ou=users,dc=ldapserver,dc=new"
AUTH_LDAP_UID_FIELD = "uid"
AUTH_LDAP_BIND_USER = "uid=rrscw,ou=users,dc=ldapserver,dc=new"
AUTH_LDAP_BIND_PASSWORD = "rrscw"

### Environment

OS: Ubuntu Server 20.04
Python on VM: 3.8.10

### Checklist

Make sure to follow these steps before submitting your issue - thank you!

- [ ] I have checked the superset logs for python stacktraces and included it here as text if there are any.
- [ ] I have reproduced the issue with at least the latest released version of superset.
- [ ] I have checked the issue tracker for the same issue and I haven't found one similar.

**Comment by villebro** [comment]
I am moving this to Discussions, as this isn't specifically reporting a bug, but requesting configuration help.
